package.json

```diff
@@ -23,6 +23,7 @@
   },
   "homepage": "https://github.com/adobe/helix-shared#readme",
   "dependencies": {
+    "jsdom": "^13.0.0",
```

Comment:
If you find a way of eliminating the need to depend on jsdom as runtime dependency, that would be great. It's pretty large library (3.4mb).

Reply:
The tests definitely need JSDOM; I am afraid there is no way to get around that.
The code depends on being passed a dom node, so this pulls in a dependency for jsdom implicitly, the only reason for requiring jsdom outside tests is getComputedStyle (which is creating issues when being used from other repos anyways).
I would prefer using https://developer.mozilla.org/en-US/docs/Web/API/Element/computedStyleMap, but unfortunately this is not available in jsdom.
The only alternatives I can think of is (1) lazyload jsdom in getComputedStyle (2) use JSDOM internals to find the window of a dom element (defaultView won't work since it returns null for detached elements)

Reply:
ok. thanks for the explanation. you are of course correct with the implicit dependency. 
An alternative approach would probably be, to offer this code to the **jsdom** library if this is desired.

Reply:
Pushing it down into `devDependencies` should work, shouldn't it?

Reply:
not right now; getComputedStyles is the blocker…

Reply:
Found a workaround…we just have to disallow cloned document nodes…